Libraries/ReactNative/AppContainer.js

```diff
@@ -35,6 +33,10 @@ type State = {|
   hasError: boolean,
 |};
 
+type ContextType = {rootTag: number, ...};
```

Comment:
You should remove the ... since we know this is exact now since it’s new

Reply:
Also, should probably wrap the object in $ReadOnly<> for good measure, if flow is okay with it.